code/modules/mob/living/carbon/human/species/station/traits_vr/neutral.dm

```diff
@@ -81,4 +81,16 @@
 	name = "Brutal Predation"
 	desc = "Allows you to tear off limbs & tear out internal organs."
 	cost = 0 //I would make this cost a point, since it has some in game value, but there are easier, less damaging ways to perform the same functions.
-	var_changes = list("hard_vore_enabled" = 1) //The verb is given in human.dm
\ No newline at end of file
+	var_changes = list("hard_vore_enabled" = 1) //The verb is given in human.dm
+
+/datum/trait/trashcan
+	name = "Trash Can"
+	desc = "Someone thought it'd be hilarious to eat the snack wrappings too instead of having to look for a bin or just littering like an animal."
+	cost = 0
+	var_changes = list("trashcan" = 1)
+
+/datum/trait/hardfeet
+	name = "Hard Feet"
+	desc = "Do you feel like your nice clawed, scaled, hooved, armored, or otherwise just hella calloused feet need more freedom without the fear of glass shards?"
+	cost = 0
+	var_changes = list("flags" = NO_MINOR_CUT) //Checked the flag is only used by shard stepping.
```

Comment:
Does this change their entire "flags" variable to NO_MINOR_CUT or does it just add it?

Reply:
Hmm I think it changes it. Tried to get it working as addition but couldn't figure it out. Took a deeper look at how the custom species add things and found out they start with empty flags anyway and don't even copy them from the selected base species.

Reply:
Alright. If that's the case, then :+1: since none of those species listed can use traits and we  have no other traits that change the flags var. Checked species.dm and flags = 0 by default.

Reply:
This should cost just one trait point.